go.mod

```diff
@@ -22,13 +22,12 @@ require (
 	github.com/rook/rook v1.2.4
 	github.com/stretchr/testify v1.4.0
 	go.uber.org/zap v1.13.0
-	google.golang.org/appengine v1.6.1 // indirect
 	k8s.io/api v0.0.0-20191005115622-2e41325d9e4b
 	k8s.io/apiextensions-apiserver v0.0.0-20190820104113-47893d27d7f7
 	k8s.io/apimachinery v0.0.0-20191005115455-e71eb83a557c
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/cloud-provider v0.0.0-20191114015453-79225fba1e26 // indirect
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
```

Comment:
Is this related to nooba changes?

Reply:
yes. it was updated with `make deps-update`